app/components/marketing/hero-section.tsx

```diff
@@ -39,21 +40,21 @@ export default function HeroSection() {
                   type="text"
                   id="q"
                   name="q"
-                  className="block w-full py-4 pl-10 text-md text-gray-900 md:text-center border border-gray-300 rounded-l-lg  bg-gray-50 focus:ring-2 focus:outline-none focus:ring-blue-500"
-                  placeholder='"Frisbee in San Fran"'
+                  className="block w-full py-4 pl-10 text-xs sm:text-base text-gray-900 md:text-center lg:text-2xl border border-gray-300 rounded-l-lg italic bg-gray-50 focus:ring-2 focus:outline-none focus:ring-blue-500"
+                  placeholder='"Frisbee in San Francisco"'
```

Comment:
For the people who are not familiar with contractions like San Fran. It was mentioned in one of the sessions 🙂